debian/control

```diff
@@ -10,6 +10,7 @@ Build-Depends:
 	libgtop2-dev,
 	libjson-glib-dev,
 	libostree-dev,
+	libsystemd-dev,
```

Comment:
I think here is `systemd-dev` to fix the build failure:
```
Run-time dependency systemd found: NO (tried pkgconfig)

../../meson.build:31:0: ERROR: Dependency "systemd" not found, tried pkgconfig
```

Reply:
Quite right.

Reply:
I even got it right in the commit message! 😁